src/catalogue/items/catalogueItemsTable.component.test.tsx

```diff
@@ -482,11 +482,7 @@ describe('Catalogue Items Table', () => {
     const rowToggleSelect = screen.getAllByLabelText('Toggle select row');
     await user.click(rowToggleSelect[1]);
 
-    await waitFor(() => {
-      expect(
-        screen.getByRole('button', { name: 'Move to' })
-      ).toBeInTheDocument();
-    });
+    expect(screen.getByRole('button', { name: 'Move to' })).toBeInTheDocument();
```

Comment:
Ah yes, I originally thought it wouldn't need it because the documentation I found all said that userEvent.click already had a wait for dom changes (although not always long enough if there is a useEffect). The examples however always uses find afterwards, so I guess that is preferred after all. Not sure about the added tests now though - they work currently without any flakiness because of the wait in await, but if we truly expect it to not exist originally perhaps they themselves may not always work.

Reply:
The tests you have added are fine.